examples/container-with-vitest/README.md

```diff
@@ -0,0 +1,11 @@
+# Astro + [Vitest](https://vitest.dev/) Example
+
+```sh
+npm create astro@latest -- --template with-vitest
```

Comment:
```suggestion
npm create astro@latest -- --template container-with-vitest
```

Reply:
Probably that the rest of the README needs to be updated too, but at least making the code snippet correct is the most important 😅